Gruntfile.coffee

```diff
@@ -802,52 +802,27 @@ gruntConfig = {
             'freedom-social-wechat': '<%= pkgs.freedomwechat.version %>'
             'freedom-social-quiver': '<%= pkgs.freedomquiver.version %>'
         }]
-  #-------------------------------------------------------------------------
-  # All typescript compiles to locations in `build/`
-  # Typescript compilation rules
+
+  # One pass for code running inside freedom.js modules and another
+  # for code running outside, due to the differences in the meaning
+  # of the (global) freedom object between the two environments.
   ts:
-    # Compile all non-sample typescript code into the development build
-    # directory.
-    devInModuleEnv: compileTypescript [
-      devBuildPath + '/lib/**/*.ts'
+    moduleEnv: compileTypescript [
+      devBuildPath + '/**/*.ts'
       devBuildPath + '/interfaces/**/*.ts'
       devBuildPath + '/generic_core/**/*.ts'
```

Comment:
I think I'm a tiny bit confused here, why does this need to be defined in addition to the `**/*.ts` rule?